dockerfiles/init/manifests/che.env

```diff
@@ -429,7 +429,13 @@ CHE_SINGLE_PORT=false
 #     property 'che.workspace.auto_snapshot=false'.
 #CHE_WORKSPACE_AUTO__SNAPSHOT=false
 
-#SYSTEM_SUPER_PRIVILEGED_MODE=false
+# System Super Privileged Mode
+#     Grants users with the manageSystem permission additional permissions for
+#     getByKey, getByNameSpace, stopWorkspaces, and getResourcesInformation.
```

Comment:
It'd be better to remove information about ResourcesService while it is not moved to che.

Reply:
it sill be moved soon as well

Reply:
Thanks. Then OK to leave it